app/javascript/components/lists/ListGroup.js

```diff
@@ -27,6 +27,11 @@ const ListGroup = (props) => {
     })
   }
 
+  const setState = () => {
+    // console.log("State Set");
+    // window.location.reload();
+  }
```

Comment:
This lambda doesn't appear to be doing anything but you're passing it around.  Is this a remnant of no-longer-needed cruft?